packages/pn-personafisica-webapp/src/component/Contacts/__test__/SpecialContacts.test.tsx

```diff
@@ -15,6 +17,19 @@ jest.mock('react-i18next', () => ({
   Trans: (props: { i18nKey: string }) => props.i18nKey,
 }));
 
+
```

Comment:
Se i componenti di errore vengono testati in pn-commons (come fatto per altri componenti in comune) non sarà necessario testare la casistica (api error) qui e negli altri componenti esterni a pn-commons